components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/action/model/ClaimPathInfo.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth.action.model;
+
+/**
+ * This class is used to identify the claim and the index of the value of the claim to be updated in the
+ * {@link AccessToken}. The claim value of the {@link AccessToken} is an array in this case.
+ */
+public class ClaimPathInfo {
```

Comment:
I think ```Claim``` may be applicable class name, given that attributes explain the claim in the AT

Reply:
This is not a claim. Basically this rather identifies the claim name in the path redefined at a particular operation in response.

e.g:

{
      "op": "replace",
      "path": "/accessToken/claims/aud/-",
      "value": "example.com"
    }

Here when above operation is processed, this class is used to identify the claim name in the path which is "aud" and the index that's defined with respect to that claim which is "-" that denotes the last index